src/transformState.js

```diff
@@ -6,6 +6,26 @@
  */
 function transformState(state, actions) {
   // write code here
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          delete state[key];
+        };
+        break;
+
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
```

Comment:
Redundant break statement, you don't have any cases after this one

Reply:
but if there will be added a new case statement in the future. There is a chance to forget to add `break` to the previous one😉

Reply:
It is like a “trailing” or “hanging” comma in object declaration. Makes it easier to add/remove/move around properties ("cases"), because all lines become alike.

Reply:
Agree with Alina. On my opinion, it is better to leave it that way.

Reply:
Then you should one, remember about `default:`